tests/Util/LocaleCodeProviderTest.php

```diff
@@ -20,13 +21,16 @@
 #[Package('checkout')]
 class LocaleCodeProviderTest extends TestCase
 {
+    private MockObject $logger;
```

Comment:
I'm surprised that phpstan don't complain:
```suggestion
    private LoggerInterface&MockObject $logger;
```